addons/survey/views/survey_question_views.xml

```diff
@@ -29,43 +27,51 @@
                             <field name="question_type" widget="radio" attrs="{'required': [('is_page', '=', False)]}" />
                         </group>
                         <group>
-                            <div class="col-lg-6 offset-lg-3 o_preview_questions">
+                            <div class="col-lg-6 offset-lg-3 w-100 o_preview_questions">
+                                <!-- Multiple choice: only one answer -->
+                                <div attrs="{'invisible': [('question_type', '!=', 'simple_choice')]}" role="img" aria-label="Multiple choice with one answer" title="Multiple choice with one answer">
```

Comment:
Linting (for this whole PR): when having long lines, you may define as multi-line to ease reading, like
```
<div attrs="{'invisible': [('question_type', '!=', 'simple_choice')]}" role="img"
     aria-label="Multiple choice with one answer"
     title="Multiple choice with one answer">
    <i>...</i>
</div>
```